schedule/sles4sap/sap_deployment_automation_framework/sdaf_cleanup.yml

```diff
@@ -9,7 +9,4 @@ schedule:
   - sles4sap/sap_deployment_automation_framework/connect_to_deployer
```

Comment:
Just a note: While merging this, the cleanup test needs to be removed from Job group since cleanup module is in schedule above.

Reply:
Yes. I will do this in following MR.